src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -489,6 +489,7 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
                                 var fixerMetadata = TryGetMetadata(fixer);
 
                                 using (addOperationScope(fixerName))
+                                using (Logger.LogBlock(FunctionId.CodeFixes_GetCodeFixesAsync, KeyValueLogMessage.Create(LogType.UserAction, m => CreateLogProperties(m, fixer)), cancellationToken))
```

Comment:
what does LogType.UserActio mean?

Reply:
I think it is finally passed into the telemetry api
https://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Telemetry/TelemetryLogger.cs,43
And indicate it is a user action.
https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.telemetry.usertaskevent?view=visualstudiosdk-2022
But I don't know is there any effect in the data we really see.
@ryzngard @jmarolf Do you guys know about this?

Reply:
AFAIK we don't directly use it in any of our queries, but it gets reported with "entitytype" of "UserTask".

Reply:
Tested it locally, as Andrew said the event is UserTask (User Action) and Operation (Trace)